3581/3/app/views/comment_modal_form.erb

```diff
@@ -0,0 +1,41 @@
+<div class="modal fade" id="comment" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle" aria-hidden="true">
+  <div class="modal-dialog modal-dialog-centered" role="document">
+    <div class="modal-content" style="background-color: #FFE4B5">
+      <div class="modal-header">
+        <h5 class="modal-title" id="exampleModalLongTitle">Review</h5>
+        <button type="button" class="close" onclick="document.getElementById('addForm').reset()" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">                
+        <div class='row'>
+          <div class='col-md-1'>
+            <button class="btn btn-success btn-sm" disabled>
+              <span class="fas fa-thumbs-up" ></span>
+            </button>
+          </div>
+          <div class="col">
+            <select class="form-control mb-3" style='width: 80px;' name='grade'>
+              <option>1 ⭐️</option>
```

Comment:
Could you explain exactly what do you want? Because in view you can choose digits 1 till 5, and nothing more, because it's select. I've already added validation of range in model.